common/changes/@microsoft/rush/ianc-fix-time-printing_2018-05-02-19-19.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@microsoft/rush",
+      "type": "none"
+    }
+  ],
+  "packageName": "@microsoft/rush",
+  "email": "iclanton@users.noreply.github.com"
+}
```

Comment:
This since `rush build` is a pretty important step, perhaps it should defensively start by setting the process.exitCode to 1 and then reassign it to 0 only if the build succeeds.  The issue we had today was pretty bad.